net.sourceforge.DuneLegacy.yaml

```diff
@@ -24,13 +24,12 @@ modules:
     sources:
       - type: git
         url: https://git.code.sf.net/p/dunelegacy/code
-        tag: v0.96.4
-        commit: dede956fbd6f6f6bbee98a059c7edb677e67fc6d
+        commit: 6ea9ac96854daa8c75ba429e78dc6716b147e106
```

Comment:
We need a released version. Ask upstream for a non-Windows release. https://dunelegacy.sourceforge.net/website/news.html#2022-12-27-00-00-00